src/main/java/com/redhat/quarkus/mandrel/collector/report/endpoints/ImageStatsResource.java

```diff
@@ -92,6 +91,23 @@ public ImageStats[] getByTag(@PathParam("tag") String tag) {
         return collection.getAllByTag(tag);
     }
 
+    // TODO: Not sure about the API, we need some consistency...
```

Comment:
Please remove this comment. Either fix it right away (make it consistent?), or drop it.

Reply:
O.K., I'll unite all under `lookup`. If it turns out to be clumsy, we refactor later...

Reply:
@jerboaa The lookup Api usage is outlined in the `ImageStatsResourceTest#testRunnerInfoLookup`.